scripts/auto_rerun_test.sh

```diff
@@ -15,7 +15,7 @@ ulimit -a
 
 #pleanse ensure tow test command's argument is same.
 RUST_LOG=OFF RUST_BACKTRACE=0 cargo xtest --exclude starcoin-move-prover -j 15 -- --test-threads=10 --color never --format pretty |tee "$TEST_RESULT_FILE" ||true
-grep -e '^test[[:space:]][^[:space:]]*[[:space:]]\.\.\.[[:space:]]FAILED' "$TEST_RESULT_FILE" >"$TEST_RESULT_FAILED_FILE" ||true
+grep -e '^test[[:space:]][^[:space:]]*[[:space:]]\+\.\.\.[[:space:]]FAILED' "$TEST_RESULT_FILE" >"$TEST_RESULT_FAILED_FILE" ||true
```

Comment:
use `[[:space:]]\+` to match multiple space output of `cargo test --package starcoin-transactional-test-harness --test tests` like

    test run_test::cases/arg_variable.move         ... FAILED